packages/storybook/templates/cxl-minidegree-progress-widget.js

```diff
@@ -0,0 +1,82 @@
+import { html } from 'lit-html';
+
+export default html`
+<cxl-minidegree-progress-widget id="post-12067">
```

Comment:
We should have all templates indented the same, see `templates/cxl-institute-content-header.js` right above.

It will create a bigger diff in the commit, but that doesn't matter right now, as a bunch of things will get squashed anyway.